stores/rocksdb/src/commonMain/kotlin/kotlinx/document/database/rocksdb/RocksdbDataStore.kt

```diff
@@ -26,7 +69,11 @@ public class RocksdbDataStore(private val delegate: RocksDB) : DataStore {
         delegate.deletePrefix(name)
     }
 
-    override fun close() {
+    override suspend fun commit() {
+        withContext(Dispatchers.IO) { delegate.flushWal(true) }
+    }
+
+    override suspend fun close() {
```

Comment:
Wrap it with `withContext(Dispatcher.IO)` and close the `CoroutineScope` if needed.